support-workers/src/main/scala/com/gu/stripe/StripeService.scala

```diff
@@ -59,3 +66,30 @@ class StripeServiceForCurrency(val stripeService: StripeService, currency: Curre
   val getPaymentMethod = com.gu.stripe.getPaymentMethod.apply(this)_
 
 }
+
+object StripeError {
```

Comment:
this is duplicated across 2 files

Reply:
thanks yes, the whole "Stripe service" is.  In one side of my mind I wanted to go full cohesive refactor but at the moment things are more grouped by type than function.
I don't think the stripe service has any expensive dependencies of its own, but I briefly tried to move it to the new support-rest project, which they both already depend on.  However I think I got tripped up with some dependency on Stripe config.  I will have another go, as I don't think stripe client stuff has any fundamentally expensive dependencies that we don't want in support-rest.

Reply:
ok this is done, it wasn't too bad.  Stripe service is still duplicated in name, but in reality it's configured in a different way and still shares the web service helper.